blog/2024-08-19-join-us-for-wasmcloud-innovation-day/index.mdx

```diff
@@ -0,0 +1,75 @@
+
+We wanted to get the open source community together to dive into the latest innovation, hear the experiences of wasmCloud users and customers, and celebrate the work of our maintainers and community contributors. That’s why we created **wasmCloud Innovation Day**, taking place on the 18th September from 10am ‘til 6pm ET.
+
+Whether you’re already using Wasm or a complete newbie, this one-day virtual event is the ideal immersion in all things WebAssembly and wasmCloud. We’ll explore wasmCloud’s technical architecture, fresh and growing feature set and, most importantly, hear how companies in a host of industries are working with this popular open source CNCF project.
+
+Here’s what to expect—check out the full agenda below and [register now](https://wasmcloud.com/innovation-day)!
+
+**wasmCloud in Industry:** Hear how engineers from the world’s largest organizations are using wasmCloud in real-world use cases…stay tuned for details.
+
+**Technical Deep Dive:** wasmCloud has been transformed with WASI 0.2 and wasmCloud 1.1 is full of features designed to make building with components a joy. From secrets support to creating your own custom capabilities, we’ll take a deeper look at what’s new, and how to get the most out of 1.1.
+
+**Community Workshop and Hangout:** Join us in Gather space to build your own Wasm application and hang with industry leaders, our valued open source contributors and wasmCloud users.
+
+## **Agenda:**
+
+**10 - 11 ET: wasmCloud Weekly Community Meeting**
+
+- In a special edition of our [weekly community meeting](https://wasmcloud.com/community), we’ll focus on the essential role of open source maintainers, exploring what being a maintainer entails, and how you can be a part of the project.
+
+**11 - 11.15: Opening Keynote:** **The Pitfalls of Modern Platform Engineering: [Bailey Hayes](https://github.com/ricochet)**
+
+- Learn how to end the ‘rinse and repeat’ of application development, improve life with Kubernetes, and run distributed workloads securely and efficiently.
+
+**11.15 - 12.15: wasmCloud 1:1, Technical Deep Dive**
+
+- **Secrets**
+    - [**Brooks Townsend**](https://github.com/brooksmtownsend) explains why this is the most requested feature amongst our enterprise customers and partners and digs into how secrets work in wasmCloud.
+- **Evolution in Golang**
+    - [**Lucas Fontes](https://github.com/lxfontes)** and **[Brooks Townsend](https://github.com/brooksmtownsend)** discuss ****the ****Go provider SDK and building Go components with TinyGo.
```

Comment:
Relocate asterisks inside brackets

I have some erroneous asterisks later in the sentence there before and after 'the' ..Go provider SDK

Reply:
Thanks for the catch here, addressed in the next commit